react/components/EmptySearch.js

```diff
@@ -12,9 +12,21 @@ import styles from '../index.css'
 import emptyStyles from './EmptySearch.css'
 import { getShipsTo } from '../utils/AddressUtils'
 import GPSDenied from '../assets/components/GPSDenied'
-import { ERROR_NOT_ALLOWED } from '../constants'
+import { ERROR_COULD_NOT_GETLOCATION, ERROR_NOT_ALLOWED, ERROR_NOT_FOUND } from '../constants'
 import { injectState } from '../modalStateContext'
 
+const getGeolocationErrorMessage = (status) => {
+  switch (status) {
+    case ERROR_NOT_ALLOWED:
+      return 'askGeolocationDenied'
+    case ERROR_NOT_FOUND:
+    case ERROR_COULD_NOT_GETLOCATION:
+      return 'errorCouldNotGetLocation'
```

Comment:
This message reads "Unable to identify your current location", which is good for this case I thing (unknown geolocation failure or timeout)